cmd/meroxa/root/apps/deploy.go

```diff
@@ -535,56 +537,69 @@ func (d *Deploy) validateAppJSON(ctx context.Context) error {
 		return err
 	}
 
-	if d.appConfig == nil {
-		d.lang, err = turbine.GetLangFromAppJSON(ctx, d.logger, d.path)
-		if err != nil {
-			return err
-		}
-		d.configAppName, err = turbine.GetAppNameFromAppJSON(ctx, d.logger, d.path)
-		if err != nil {
-			return err
-		}
-		config, err = turbine.ReadConfigFile(d.path)
-		d.appConfig = &config
-		if err != nil {
-			return err
-		}
+	// exit early if app config is loaded
+	if d.appConfig != nil {
+		return d.validateLanguage()
```

Comment:
We need to validate language after `GetLangFromAppJSON` since that's where we assign language.

Reply:
This only happens if `appConfig` is nil, as per https://github.com/meroxa/cli/blob/eae9dc253cf508b008b028e11db87b3ea1276be4/cmd/meroxa/root/apps/deploy.go#L538-L557
Was the original code wrong?

Reply:
@lyuboxa I see it. Yeah, I think it's fine the way it is now.